Backend/src/main/java/ktsnvt/tim1/model/RegisteredUser.java

```diff
@@ -9,6 +11,7 @@
 @Entity
 public class RegisteredUser extends User {
     @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL, mappedBy = "registeredUser")
+    @JsonIgnore
```

Comment:
What is the purpose of @JsonIgnore here? If problem is LazyException, delete the annotation and put @Transactional above the method getUser_userLoggedIn_userReturned.

Reply:
I agree with @MasterRadule